rnr_control/src/rnr_control.c

```diff
@@ -719,6 +877,192 @@ static void handle_arguments( ps_node_ref const node_ref, int argc, char **argv
 }
 
 
+//
+static int send_basic_command(
+        ps_node_ref node_ref,
+        const ps_command_id cmd,
+        const ps_guid dest_guid,
+        ps_command_msg * const cmd_msg )
+{
+    int ret = DTC_NONE;
+    unsigned long dest_sdf_id = 0;
+    ps_node_type dest_node_type = 0;
+    ps_identifier dest_node_id = 0;
+
+
+    // break down the destination GUID
+    (void) psync_guid_get_sdf_id( dest_guid, &dest_sdf_id );
+    (void) psync_guid_get_node_type( dest_guid, &dest_node_type );
+    (void) psync_guid_get_node_id( dest_guid, &dest_node_id );
+
+    printf( "\n" );
```

Comment:
All these prints were a relic of the example application the code was copied from. in the `rnr_control` example/tool we don't care about these prints.